crates/sui-core/src/consensus_adapter.rs

```diff
@@ -506,7 +500,7 @@ pub struct CheckpointConsensusAdapter {
     /// The network client connecting to the consensus node of this authority.
     consensus_client: TransactionsClient<sui_network::tonic::transport::Channel>,
     /// Channel to request to be notified when a given consensus transaction is sequenced.
-    tx_consensus_listener: Sender<ConsensusListenerMessage>,
+    _tx_consensus_listener: Sender<ConsensusListenerMessage>,
```

Comment:
Why prepending with `_`?

Reply:
I am being lazy in this PR - `tx_consensus_listener` is no longer used in adapter, but I kept it for now.
Will remove in the next PR when I will be removing ConsensusListener

Reply:
Sounds good. Later certificate handlers will be sending certificates to transaction manager and use `effects_notify_read()` to wait on results. It may avoid the step to wait on consensus as well.

Reply:
Huh, I did not think about it! Indeed, this sounds reasonable